### PR TITLE
PLAT-432: release semaphore for outgoing

### PR DESCRIPTION
**Pull request by aalekseev-ru**
Release global semaphore while waiting outgoing result.
Reacquire it when getting result.

### CODE REVIEW

File: ledger-core/virtual/execute/execute.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ type SMExecute struct {
 	messageSender         messageSenderAdapter.MessageSender
 	pulseSlot             *conveyor.PulseSlot
 	authenticationService authentication.Service
+	globalSem             conveyor.ParallelProcessingLimiter
```
Comment (ita-sammann) [inline ledger-core/virtual/execute/execute.go:68]: Should it be `RunnerLimiter`?

File: ledger-core/virtual/execute/execute.go [hunk 2]
```diff
@@ -65,6 +66,7 @@ type SMExecute struct {
 	messageSender         messageSenderAdapter.MessageSender
 	pulseSlot             *conveyor.PulseSlot
 	authenticationService authentication.Service
+	globalSem             tool.RunnerLimiter
```
Comment (bigbes) [inline ledger-core/virtual/execute/execute.go:69]: please, rename it to `globalSemaphore`

File: ledger-core/virtual/execute/execute.go [hunk 3]
```diff
@@ -67,6 +68,7 @@ type SMExecute struct {
 	messageSender         messageSenderAdapter.MessageSender
 	pulseSlot             *conveyor.PulseSlot
 	authenticationService authentication.Service
+	globalSemaphore       tool.RunnerLimiter
```
Comment (sexyhamster) [inline ledger-core/virtual/execute/execute.go:71]: maybe call it executionLimiter or something like that? semaphore with type that is not semaphore, when we have semaphore as different type.

Comment (aalekseev-ru) [reply ledger-core/virtual/execute/execute.go:71]: see conversation above, renamed it after @bigbes  request

**Pull request merged**